ceph_deploy/util/pkg_managers.py

```diff
@@ -206,6 +210,18 @@ def clean(self):
         """Clean metadata/cache"""
         raise NotImplementedError()
 
+    def add_repo_gpg_key(self, url):
+        """Add given GPG key for repo verification"""
+        raise NotImplementedError()
+
+    def add_repo(self, name, url, gpg_url):
```

Comment:
to avoid the problem of future backward incompatibility I would just slap a `**kw` here and pass in name, url, gpg_url, and anything else as keyword arguments. That way you can use what you are being passed in today, but you are allowing passing more later without having to change the signature (and possibly making it look odd)